sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/ExpressionLanguageInjection.kt

```diff
@@ -44,6 +44,26 @@ object ExpressionLanguageInjection {
             targetClassName = "javax.el.ExpressionFactory",
             targetMethod = "createMethodExpression",
         ),
+        MethodHook(
+            type = HookType.BEFORE,
+            targetClassName = "org.apache.el.ExpressionFactoryImpl",
```

Comment:
According to https://tomcat.apache.org/tomcat-8.0-doc/api/org/apache/el/ExpressionFactoryImpl.html, this class inherits from `javax.el.ExpressionFactory`, which we already hook with the annotations above. Hooks descend to subclasses, so this Apache-specific hook shouldn't be needed.

Could you check whether you can drop it and still get findings?

Reply:
Yes you are right, I can still get the findings without the redundant ExpressionFactoryImpl. I already update my commit, and thank you for your insights for remove the redundant code!